pages/treeherder.py

```diff
@@ -49,6 +51,15 @@ def unchecked_repos(self):
     def unclassified_failure_count(self):
         return int(self.find_element(*self._unclassified_failure_count_locator).text)
 
+    def clear_filter(self):
```

Comment:
+1 on removing this if it's not used
-1 on renaming to add 'click' if we decide to use/keep it